### PR TITLE
Decathlon user story

### PR DESCRIPTION
**Pull request by loicmathieu**
** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc **

### CODE REVIEW

File: _posts/2021-09-14-decathlon-user-story.adoc [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+
+=== __Victor Gallet - Senior Developer and Kafka Expert__
+
+When I joined the team, around ten microservices had already been deployed with Quarkus.
+I had not yet had the opportunity to play with Quarkus, I had just seen presentations at a meetup and the framework interested me. Having worked since the beginning of my career with the Spring framework, I wanted to discover the big differences compared to Quarkus.
+Despite utility classes and different CDI annotations, I was able to get up and running with Quarkus very quickly. The Quarkus Guides have been a huge help as they focus on one topic and get straight to the point. For example, since our microservices communicate exclusively with Apache Kafka, the Quarkus integration guide with Apache Kafka allowed me to immediately understand the concepts and our different components.
+
+Compared to the dependency injection and the implementation provided by Quarkus (ArC), I had no problem navigating it. The concepts are standard and I only had to discover some new annotations like @ApplicationScoped and @Singleton to name just the most used ones.
```
Comment (cescoffier) [inline _posts/2021-09-14-decathlon-user-story.adoc:117]:
```suggestion
Compared to the dependency injection and the implementation provided by Quarkus (ArC), I had no problem navigating it. The concepts are standard and I only had to discover some new annotations like `@ApplicationScoped` and `@Singleton` to name just the most used ones.
```

File: _posts/2021-09-14-decathlon-user-story.adoc [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+            });
+}
+----
+
+1. The method takes a stream of Kafka messages.
+2. First, we group by payload type.
+3. Then, we group by batch of size messages with a max duration of duration.
+4. Then, we remove empty batches.
+5. Then, we map each batch of message to a new format.
+6. Then we return a Kafka message with the batch in it.
```
Comment (cescoffier) [inline _posts/2021-09-14-decathlon-user-story.adoc:186]: The <6> anchor in the code is misplaced. It should be at the end of a line.

File: _posts/2021-09-14-decathlon-user-story.adoc [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+Pretty straightforward right ?
```
Comment (cescoffier) [inline _posts/2021-09-14-decathlon-user-story.adoc:188]:
```suggestion
Pretty straightforward, right?
```

File: _posts/2021-09-14-decathlon-user-story.adoc (hunk 2 above)
Comment (loicmathieu) [reply _posts/2021-09-14-decathlon-user-story.adoc:185]: Fixed, thanks

**Comment by maxandersen** [comment]
@loicmathieu please mark ready for review when you are ok to post it. We can post it as soon as monday - then just need to set date to 2021-09-13 and I'll push it monday.

**Pull request merged**